custom_components/bambu_lab/camera.py

```diff
@@ -0,0 +1,80 @@
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.core import HomeAssistant
+from yarl import URL
+from homeassistant.components import ffmpeg
+
+from .const import DOMAIN, LOGGER
+from .models import BambuLabEntity
+from .pybambu.const import Features
+
+from homeassistant.components.camera import Camera, CameraEntityFeature
+
+from .coordinator import BambuDataUpdateCoordinator
+
+
+async def async_setup_entry(
+        hass: HomeAssistant,
+        entry: ConfigEntry,
+        async_add_entities: AddEntitiesCallback
+) -> None:
+    coordinator: BambuDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    if coordinator.get_model().supports_feature(Features.CAMERA_RTSP):
+        entities_to_add: list = [BambuLabCamera(coordinator, entry)]
+        async_add_entities(entities_to_add)
+
+
+class BambuLabCamera(BambuLabEntity, Camera):
+    """ Defined the Camera """
+
+    _attr_translation_key = "camera"
+    _attr_icon = "mdi:camera"
+    _attr_supported_features = CameraEntityFeature.STREAM
+    _attr_brand = "Bambu Lab"
+
+    def __init__(
+            self,
+            coordinator: BambuDataUpdateCoordinator,
+            config_entry: ConfigEntry
+    ) -> None:
+        """Initialize the sensor."""
+
+        self._attr_unique_id = f"{config_entry.data['serial']}_camera"
```

Comment:
For consistency with other devices should we prepend this with the model name?

Reply:
Never mind we're already inconsistent and this is unique ID so the model number is irrelevant. Good to go on this.